api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -734,3 +737,14 @@ message CustomInlineHeader {
   // The type of the header that is expected to be set as the inline header.
   InlineHeaderType inline_header_type = 2 [(validate.rules).enum = {defined_only: true}];
 }
+
+message MemoryAllocatorManager {
+  // Configures tcmalloc to perform background release of free memory in amount of bytes per ``memory_release_interval`` interval.
+  // If not configured, no memory release will occur.
```

Comment:
Do we need to differentiate between 0 and unset?  If so, we should use `google.protobuf.UInt64Value` instead of `uint64`.  If not, this comment should be changed to say "if 0" instead of "if not configured".

Reply:
@markdroth i added more clarification like this:
```
// If equals to ``0``, no memory release will occur. Defaults to ``0``.
```